make/go/dep_golangci_lint.mk

```diff
@@ -7,9 +7,9 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://github.com/golangci/golangci-lint/releases 20230219 checked 20230222
+# https://github.com/golangci/golangci-lint/releases 20230318 checked 20230320
 # Check for new linters and add to .golangci.yml (even if commented out) when upgrading
-GOLANGCI_LINT_VERSION ?= v1.51.2
+GOLANGCI_LINT_VERSION ?= v1.52.0
```

Comment:
All of the linters have been updated to fix issues w/ generics now: https://github.com/golangci/golangci-lint/issues/2649#issuecomment-1469189458